lib/peeps.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
+# class for peep object
+class Peeps
+
+  # determines whether test or development database is accessed
+  @database = if ENV['ENVIRONMENT'] == 'test'
+                p "Establishing Test environment"
+                PG.connect(dbname: 'Chitter_test')
+              else
+                PG.connect(dbname: 'Chitter')
+              end
```

Comment:
I'm sure that you would've gotten to this step with more time, but setting up a separate database connection means you get to replace all of this with two lines of code, which is tidier looking.